chapters/daemon/index.rst

```diff
@@ -0,0 +1,6 @@
+Jina Daemon API Specification
+=============================
+
+[Link to Jina Daemon OpenAPI Specification](https://api.jina.ai/daemon/)
```

Comment:
@rutujasurve94 this markdown style link won't work, please use rst link: `Link text <link URL>`_